index.ts

```diff
@@ -0,0 +1,7 @@
+import { FastifyTypeProvider } from 'fastify'
+
+import { Static, TSchema } from '@sinclair/typebox'
+
```

Comment:
We should either export all the helper from `typebox` or move `typebox` as `devDependencies`.

```suggestion

export * from '@sinclair/typebox'

```

Reply:
I've seen it in the example from the comment, but I have removed it because I couldn't understand completely why this is needed, could you elaborate?

Reply:
I am perfectly fine if we do not plan to re-export the tools, it just convenient for the users not to import in separate module.

If `@sinclair/typebox` is not used in code (only type import is not used anyway), I think it should not be exist in `dependencies`.

Reply:
I think that I should move it to `peerDependencies` and do not re-export the tools, what do you think?